src/K9OCRS/ClientApp/src/areas/dogs/MyDogs/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+import { Button, Row, Col, Spinner } from 'reactstrap';
+import { Link } from 'react-router-dom';
+import DogCard from './components/DogCard';
+import * as actions from '../modules/actions';
+import PageHeader from '../../../shared/components/PageHeader';
+import PageBody from '../../../shared/components/PageBody';
+
+const MyDogs = (props) => {
+    const {
+        loading,
+        dogList,
+        getData,
+    } = props;
+
+    const [alerts, setAlerts] = useState([]);
+
+    useEffect(() => {
+        getData();
+    }, []);
```

Comment:
Apply this suggestion to get rid of a linting warning about the dependency array. In this case we don't need it.
```suggestion
    }, []); // eslint-disable-line
```